frontend/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "include": ["src"],
     "exclude": ["node_modules"],
     "compilerOptions": {
+        "types": ["node"],
```

Comment:
Why we need this?

Reply:
It is because I try to resolve my errors with ts compiling. But in a result, it compiles without this change, so I delete it